src/Templates/parts/nav.twig

```diff
@@ -10,6 +10,7 @@
       </button>
       {% endif %}
       <a class="navbar-brand" href="{{baseurl}}">MyRadio</a>
+      <a class="navbar-brand" href="{{homeurl}}">ury.org.uk</a>
```

Comment:
It shouldn't have 'navbar-brand' read the Bootstrap docs.
And it should probably go into the hamburger menu on small screens.